src/GovUk.Education.ExploreEducationStatistics.Data.Model.Tests/Fixtures/IndicatorGroupGeneratorExtensions.cs

```diff
@@ -34,6 +34,7 @@ public static InstanceSetters<IndicatorGroup> SetDefaults(this InstanceSetters<I
                 ig => ig.Subject,
                 (_, indicatorGroup) =>
                 {
+                    subject.IndicatorGroups ??= new List<IndicatorGroup>();
```

Comment:
We should just set a default empty list in `Subject` to avoid it being unset (and for the other properties that use lists).

Reply:
Yup happy to do that

Reply:
Slapped #nullable enable on Subject and added the defaults, ta.